### PR TITLE
chore: Link survey mobile view

### PR DESCRIPTION
**Pull request by Dhruwang**
## What does this PR do?

Fixes https://github.com/formbricks/internal/issues/62

Screenshot 2024-04-19 at 2 33 53 PM

Screenshot 2024-04-19 at 2 33 27 PM

It may break if you add too much content (long headline and description) in a question but it can be fixed with https://github.com/formbricks/formbricks/pull/2477

## How should this be tested?

Test link survey on small devices

### Required

- [x] Filled out the "How to test" section in this PR
- [x] Read [How we Code at Formbricks](<[https://github.com/formbricks/formbricks/blob/main/CONTRIBUTING.md](https://formbricks.com/docs/contributing/how-we-code)>)
- [x] Self-reviewed my own code
- [x] Commented on my code in hard-to-understand bits
- [x] Ran `pnpm build`
- [x] Checked for warnings, there are none
- [x] Removed all `console.logs`
- [x] Merged the latest changes from main onto my branch with `git pull origin main`
- [x] My changes don't cause any responsiveness issues
- [x] First PR at Formbricks? [Please sign the CLA!](https://cla-assistant.io/formbricks/formbricks) Without it we wont be able to merge it 🙏

### Appreciated

- [x] If a UI change was made: Added a screen recording or screenshots to this PR
- [x] Updated the Formbricks Docs if changes were necessary

### CODE REVIEW

**Comment by jobenjada** [comment]
Hey Dhru!

Thanks, looks good. However, if I test it on my iPhone, I have to scroll up to see the survey:

IMG_2436
IMG_2437

Other than that, I like the filling out UX much better. Really easy within the thumb range :)

**Review by jobenjada — CHANGES_REQUESTED** [review]
see comment

**Comment by Dhruwang** [comment]
Hey @jobenjada, Updated the PR 😊. For safari on mobile, there is an issue with the dynamic height of address bar, and due to which even though its `100vh`, it still scrolls because address bar at the bottom takes some height

File: packages/ui/Survey/index.tsx [hunk 1]
```diff
@@ -40,5 +40,5 @@ export const SurveyInline = (props: Omit<SurveyInlineProps, "containerId">) => {
     loadScript();
   }, [containerId, props, renderInline]);
 
-  return <div id={containerId} className="h-full w-full" />;
```
Comment (jobenjada) [inline packages/ui/Survey/index.tsx:43]: did you test this well? Feels like it'd break the layout in other places ;)

**Review by jobenjada — CHANGES_REQUESTED** [review]
Hey Dhru!

Noticed two more things:

The bottom border corners shouldn't be rounded on mobile:

image

Secondly, we currently have uneven paddings for top and bottom (`pt-6` and `pb-3`) which is good for Desktop and in-app. 

On mobile, pls sync them so its `pt-6` and `pb-6`

So should look like this:

image

Thanks a lot! :)

**Comment by Dhruwang** [comment]
Hey @jobenjada , have updated the PR 😊
Screenshot 2024-04-24 at 6 00 33 PM

**Pull request merged**